app/User.php

```diff
@@ -123,6 +128,7 @@ protected static function boot()
             }
 
             \DB::table('favorites')->where('user_id', $user->id)->delete();
+            \DB::table('dismissed_conferences')->where('user_id', $user->id)->delete();
```

Comment:
Import `DB` rather than referencing `\DB`

Reply:
I imported the facade.